src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -26,7 +24,7 @@
     <_hostOS>$(NETCoreSdkPortableRuntimeIdentifier.SubString(0, $(NETCoreSdkPortableRuntimeIdentifier.LastIndexOf('-'))))</_hostOS>
     <_hostArchitecture Condition="'$(OS)' != 'Windows_NT'">$(NETCoreSdkPortableRuntimeIdentifier.SubString($([MSBuild]::Add($(NETCoreSdkPortableRuntimeIdentifier.LastIndexOf('-')), 1))))</_hostArchitecture>
     <_hostArchitecture Condition="'$(OS)' == 'Windows_NT'">$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture.ToString().ToLowerInvariant)</_hostArchitecture>
-    <_hostPackageName>runtime.$(_hostOS)-$(_hostArchitecture).Microsoft.DotNet.ILCompiler</_hostPackageName>
+    <_hostPackageName>runtime.$(ToolsRID).Microsoft.DotNet.ILCompiler</_hostPackageName>
```

Comment:
This diverged from `Microsoft.DotNet.ILCompiler.SingleEntry.targets` so "Compute host package name (taken from Microsoft.DotNet.ILCompiler.SingleEntry.targets)" comment does not apply anymore. 

Maybe this can be simplified? (e.g. delete the `_hostPackageName` property and inline it in one place that it used)

Reply:
I've applied this suggestion.